database/seed.js

```diff
@@ -0,0 +1,89 @@
+const db = require('./database.js');
+const faker = require('faker');
+
+let seedProducts = function(n) {
+  for (let i = 0; i < n; i++) {
+    let temp = faker.random.words(3);
+    db.insertProduct(temp, (err, data) => {
+      if (err) {
+        console.log('error in seeding products function: ' + err);
+      }
+    })
+  }
+};
+
+seedProducts(40);
```

Comment:
I'm curious about  why seed 40 products?

Reply:
Just wanted to test it to begin with. But will change these functions around to represent 100 products